compose/service.py

```diff
@@ -679,7 +679,8 @@ def labels(self, one_off=False):
 
     def can_be_scaled(self):
         for port in self.options.get('ports', []):
-            if ':' in str(port):
+            external = split_port(port)[1]
+            if external and not (type(external) == tuple and external[1] == None):
```

Comment:
I think that if you're checking that `type(external) == tuple`, then checking it for truthiness is unnecessary. And I think [isinstance is better than comparing types](http://stackoverflow.com/a/1549854). So this can just be:

```python
if isinstance(external, tuple) and external[1] == None:
```